### PR TITLE
DLR native macos build

### PR DESCRIPTION
**Pull request by stu1130**
## Channel for questions ##
Before or while filing an issue please feel free to join our [Slack channel](https://join.slack.com/t/deepjavalibrary/shared_invite/zt-ar91gjkz-qbXhr1l~LFGEIEeGBibT7w) to get in touch with development team, ask questions, find out what's cooking and more!

## Description ##
(Brief description of what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- [ ] Code is well-documented: 
    - For user-facing API changes, Java doc has been updated. 
    - For new examples, README.md is added to explain the what the example does.
- [ ] To the my best knowledge, [examples](https://github.com/awslabs/djl/tree/master/examples) and [jupyter notebooks](https://github.com/awslabs/djl/tree/master/jupyter) are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, Java doc)
- [ ] Feature2, tests, (and when applicable, Java doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: .github/workflows/native_s3_dlr.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [macos-latest]
+    env:
+      DLR_VERSION: "1.5.0"
```
Comment (frankfliu) [inline .github/workflows/native_s3_dlr.yml:13]: I don't like have two place to store version.
I think we should only store version in gradle.properties file.
1. Use gradlew to invoke build.sh file and pass version to the shell
2. in the shell script create a folder with version name, we just copy the parent folder to s3 without need  to know the version

Comment (stu1130) [reply .github/workflows/native_s3_dlr.yml]: agreed though it is native build not jni build. I use bash tool to get the version from gradle property

**Review by frankfliu — APPROVED** [review]

File: .github/workflows/native_s3_dlr.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    strategy:
+      matrix:
+        operating-system: [macos-latest]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Clone DLR
```
Comment (lanking520) [inline .github/workflows/native_s3_dlr.yml:18]: You can use git checkout plugin in here. Refer to the TFLite build

Comment (stu1130) [reply .github/workflows/native_s3_dlr.yml:18]: right but I can't find a way to pass the env back to ref, seems like ref has to be a constant

**Review by lanking520 — APPROVED** [review]

**Pull request merged**